src/v2/Apps/Home/Components/HomeFeaturedEvents.tsx

```diff
@@ -0,0 +1,90 @@
+import { Box, Image, Text, Spacer, Shelf } from "@artsy/palette"
+import { compact } from "lodash"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { RouterLink } from "v2/System/Router/RouterLink"
+import { HomeFeaturedEvents_orderedSet } from "v2/__generated__/HomeFeaturedEvents_orderedSet.graphql"
+
+interface HomeFeaturedEventsProps {
+  orderedSet: HomeFeaturedEvents_orderedSet
+}
+
+const HomeFeaturedEvents: React.FC<HomeFeaturedEventsProps> = ({
+  orderedSet,
+}) => {
+  const events = compact(orderedSet.items).flatMap(item =>
+    item.__typename === "FeaturedLink" ? [item] : []
+  )
+
+  if (events.length === 0) return null
+
+  return (
+    <>
+      <Text variant="xl">Featured events</Text>
+
+      <Spacer mt={4} />
+
+      <Shelf>
```

Comment:
Any reason this component doesn't have `Rail` in the name?

Reply:
Ah, I think when I made it I thought it wasn't a rail until I looked later. I'll rename and merge 👍